### PR TITLE
Dashboard dev

### PR DESCRIPTION
**Pull request by kateharwood**
- Update some description text
- Re-do hiding truth plot when no evaluation data shown (now that truth plot is always shown)
- Only show WIS for forecasts that have complete quantiles
- Get rid of blank space between facets
- Make plot space bigger when more facets
- Add linetypes to plots
- Start y axis at 0
- Rename tooltip labels
- Add message about dragging to zoom

### CODE REVIEW

**Comment by kateharwood** [comment]
This is ready for review

File: dashboard/app.R [hunk 1]
```diff
@@ -111,32 +112,26 @@ ui <- fluidPage(padding=0,
                                                      "Coverage" = "coverage")),
             selectInput(
               "forecasters",
-              "Forecasters (Type a name or select from dropdown)",
+              p("Forecasters", tags$br(), tags$span("Type a name or select from dropdown", style="font-weight:normal; font-size:13px")),
```
Comment (tildechris) [inline dashboard/app.R:115]: Nit: going forward move styles into style.css

File: dashboard/app.R [hunk 2]
```diff
@@ -214,11 +223,17 @@ server <- function(input, output, session) {
...
       filter(forecaster %in% forecasters)
     
-    filteredScoreDf <- scoreDf %>% rename(Forecaster = forecaster, Date = target_end_date)
+    filteredScoreDf <- scoreDf %>% rename(Forecaster = forecaster, Week_End_Date = target_end_date)
     
     if (scoreType == "wis") {
       filteredScoreDf <- filteredScoreDf %>% rename(Score = wis)
       title = "Weighted Interval Score"
+      # Only show WIS for forecasts that have all intervals
+      filteredScoreDf = filteredScoreDf %>% filter(!is.na(`50`)) %>% filter(!is.na(`80`)) %>% filter(!is.na(`95`))
```
Comment (tildechris) [inline dashboard/app.R:232]: Nit: I think there may be a way to filter by a vector so you only have to write one filter clause.

**Review by tildechris — APPROVED** [review]
LGTM.

**Pull request merged**